### DISCUSSION TITLE
zh/games/sliding

### DISCUSSION THREAD

**Discussion by giscus** [selfboot/ai_gallery#10] [Q&A]
# zh/games/sliding

经典的在线免费数字华容道拼图游戏。支持随机生成不同难度和不同大小的起始布局，支持自定义每个格子的数字，支持获取智能解法方案！代码开源，免费在线玩！

https://gallery.selfboot.cn/zh/games/sliding

**Comment by Lightinsky523** [comment]
不儿，你这个随机有点不随机啊

**Reply by selfboot** [reply]
这里会多次出现同一局面游戏吗

**Reply by selfboot** [reply]
这里我理解了，修复了
